graphics/Coin/Portfile

```diff
@@ -1,80 +1,65 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem          1.0
-PortGroup           qt4 1.0
-PortGroup           bitbucket 1.0
+PortSystem         1.0
 
-bitbucket.setup     Coin3D coin 3.1.3 Coin-
-name                Coin
-revision            7
-license             BSD
-description         cross platform C++ OpenGL scene graph library
-long_description    Coin is a high-level 3D graphics library with a   \
-                    C++ Application Programming Interface. Coin uses scene-graph data \
-                    structures to render real-time graphics suitable for mostly all   \
-                    kinds of scientific and engineering visualization applications.
+PortGroup          cmake 1.1
+PortGroup          github 1.0
 
-conflicts           Coin-framework
-categories          graphics devel
-platforms           darwin
-maintainers         nomaintainer
+name               Coin
+version            4.0.0
+categories         graphics devel
+platforms          darwin
+revision           8
+maintainers        nomaintainer
+license            BSD
+description        cross platform C++ OpenGL scene graph library
+conflicts          Coin-framework
 
-homepage            https://bitbucket.org/Coin3D/coin/wiki/Home
+long_description   Coin is a high-level 3D graphics library with a \
+                   C++ Application Programming Interface. Coin uses scene-graph data \
+                   structures to render real-time graphics suitable for mostly all \
+                   kinds of scientific and engineering visualization applications.
 
-bitbucket.tarball_from downloads
-distname            ${name}-${version}
+# tag_prefix will be changed to v starting with version 4.0.1
+github.setup       coin3d coin ${version} Coin-
+github.tarball_from releases
 
-checksums           rmd160  e57fd6a4876a06874429f9ad6b96f166f5609362 \
-                    sha256  583478c581317862aa03a19f14c527c3888478a06284b9a46a0155fa5886d417
+homepage           https://coin3d.github.io
 
-depends_lib-append  port:simage \
-                    port:fontconfig \
-                    port:freetype \
-                    port:zlib \
-                    port:bzip2
+distname           coin-${version}-src
+distfiles          coin-${version}-src${extract.suffix}
```

Comment:
this should not be needed as the `github` PG sets it automatically

Reply:
Unfortunately the port name (Coin) and the repo name (coin) differs in case and the source tarball in the repository is added an additional `-src` suffix. So the defaults do not apply when I set `name  Coin` after the github PG.